prov/efa/src/rxr/rxr_op_entry.h

```diff
@@ -207,6 +213,33 @@ struct rxr_op_entry {
 	(*((enum rxr_x_entry_type *)	\
 	 ((unsigned char *)((pkt_entry)->x_entry))))
 
+/**
+ * @brief return the op_entry of a packet entry
+ *
+ * Each op_entry is for an send or receive operation.
+ * Note all packet entries are associated with an op entry.
+ * For example:
+ *     HANDSHAKE packets are not assoicated any send/receive operation
+ *     RMA_CONTEX packets can be associated with a rxr_read_entry.
+ *
+ * @param		pk_entry[in]		packet entry
+ * @return		pointer to the op_entry if the inpu packet entry is associated with an op
+ * 			NULL otherwise
+ */
+static inline
+struct rxr_op_entry *rxr_op_entry_of_pkt_entry(struct rxr_pkt_entry *pkt_entry)
```

Comment:
Why not name it `rxr_x_entry_of_pkt_entry` since we are extracting `pkt_entry->x_entry`?

Reply:
`x_entry` can be tx_entry, rx_entry or read_entry.

op_entry is more precise here.